updater/cmd/updater/main.go

```diff
@@ -16,14 +16,20 @@ package main
 
 import (
 	"fmt"
+	"log"
+	"os"
 
+	"github.com/observiq/observiq-otel-collector/updater/internal/download"
 	"github.com/observiq/observiq-otel-collector/updater/internal/version"
 	"github.com/spf13/pflag"
 )
 
 // Unimplemented
 func main() {
-	var showVersion = pflag.BoolP("version", "v", false, "prints the version of the collector")
+	var showVersion = pflag.BoolP("version", "v", false, "Prints the version of the collector and exits, if specified.")
+	var downloadURL = pflag.String("url", "", "URL to download the update archive from.")
+	var tmpDir = pflag.String("tmpdir", "", "Temporary directory for artifacts. Parent of the 'rollback' directory.")
+	var contentHash = pflag.String("content-hash", "", "Hex encoded hash of the content at the specified URL.")
```

Comment:
From another comment this is a hex encoded hash of a sha256? Does it come like that from BP or are we hex encoding it?

Reply:
We would hex encode it before passing it to the updater binary.

That being said, when this is moved to the collector, I assume it'll be received as a `[]byte`, and we can remove this encoding/decoding step.

Reply:
Ok, I realize this is a moot question because we're moving it to the collector but curious why we decided to hex encode it?

Reply:
Testability, basically. I can run `sha256sum` and input the output of that tool into the cli, and have it work. 

I also wasn't sure if passing arbitrary bytes into args could run into any issues, but maybe that isn't a concern. Curious if it actually does cause issues.